estela-api/api/views/project.py

```diff
@@ -178,6 +178,8 @@ def update(self, request, *args, **kwargs):
 
         serializer.save()
         headers = self.get_success_headers(serializer.data)
+        print("Project serializer: ")
+        print(serializer.data)
```

Comment:
Don't forget to remove unnecessary prints

Reply:
Done